compiler/one-cmds/utils.py

```diff
@@ -368,3 +369,32 @@ def _get_optimization_list(get_name=False):
         opt_list = [_remove_suffix(s, '.cfg') for s in opt_list]
 
     return opt_list
+
+
+def _detect_one_import_drivers():
+    """Looks for available import drivers
+
+    Returns:
+    dict: each entry is related to single detected driver,
+          key is a config section name, value is a driver name
+
+    """
+    import_drivers_dict = {}
+    bin_dir = os.path.dirname(os.path.realpath(__file__))
+    for module_name in os.listdir(bin_dir):
+        full_path = os.path.join(bin_dir, module_name)
+        if not os.path.isfile(full_path):
+            continue
+        if module_name.find("one-import-") != 0:
+            continue
+        module_loader = importlib.machinery.SourceFileLoader(module_name, full_path)
+        module_spec = importlib.util.spec_from_loader(module_name, module_loader)
+        module = importlib.util.module_from_spec(module_spec)
+        try:
```

Comment:
I'm not familiar with `importlib` library. So, I'm not sure but `try` statement is necessary? I just saw the example codes at https://docs.python.org/3/library/importlib.html#checking-if-a-module-can-be-imported and there's no that statement. 
Also, if it is necessary, why `except` statement just runs `pass`?

Reply:
I am not very experienced with `importlib` too, but this is the only way to import **arbitrary** named module I found :slightly_smiling_face:.

Problem with methods described in documentation is following: they works with ordinary named python files only (should end with "*.py").

One-cmds tools have names `one-import-tflite`, `one-import-onnx`, etc. and examples from documentation do not recognize these files as a python modules =(

Back to your question about try:
Everything above this line just initialize some structures, but do not access actual files.
Only `exec_module` call do the actual job and raise an exception in case chosen file is not a python module.

Reply:
about `pass`

I can not guarantee if there will be no other files starting with `one-import-` (maybe even unrelated to one-cmds) in current directory (especially after installation), so I decided to simply ignore import failures...

Reply:
@binarman Thank you for your explanation:) I understood `module_from_spec` method works with `*.py` as you said.